rflx/specification/parser.py

```diff
@@ -951,9 +897,81 @@ def extract_then(
                 continue
             structure.append(model.Link(source_node, *extract_then(then)))
 
+        merge_component_aspects(
+            error, component_identifier, structure, *extract_aspect(component.f_aspects)
+        )
+        merge_component_condition(
+            component_identifier,
+            structure,
+            create_bool_expression(component.f_condition, filename)
+            if component.f_condition
+            else expr.TRUE,
+        )
+
     return structure
 
 
+def merge_component_aspects(
```

Comment:
Shouldn't that be `merge_field_aspects` now (same for `_condition`)?

Reply:
It depends on if we want to be consistent to the language reference or consistent to the rest of the existing code (in this module and in the RecordFlux parser). I have opted for the latter. The optimal solution would be to refactor the code and rename "component" to "field" everywhere. I think a separate issue would be more appropriate for that.

Reply:
OK, please create one.